03 Misc/09 Right pad library/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "right-pad",
+  "version": "1.0.0",
+  "description": "A library to append text",
+  "main": "dist/rightpad.js",
+  "types": "dist/rightpad.d.ts",
```

Comment:
I don't know any `typings` in a package.json. That tells typescript where to find the library typing automagically.